# rc control
steering_min_pwm: 1100
steering_mid_pwm: 1570
steering_max_pwm: 1890

throttle_min_pwm: 1350
throttle_mid_pwm: 1500
throttle_max_pwm: 2000

mode_min_pwm: 1000
mode_mid_pwm: 1500
mode_max_pwm: 2000

servo_min: -0.34
servo_mid:  0
servo_max:  0.34

speed_min: -2.0
speed_mid:  0
speed_max:  2.0




